src/Actions/CreateBrands.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Rapidez\StatamicBrands\Actions;
+
+use Illuminate\Support\Collection;
+use Illuminate\Support\Str;
+use Statamic\Facades\Entry;
+
+class CreateBrands
+{
+    public function create(Collection $brands, string $storeCode): Collection
+    {
+        if ($brands->isEmpty()) {
+            return new Collection();
+        }
+
+        return $brands->filter(fn ($brand) => !blank($brand->name) && !blank($brand->brand_code))->map(fn ($brand) => [
+            'title' => $brand->title,
+            'brand_code' => $brand->brand_code,
+            'path' => $brand->brand_code
+        ])->each(fn ($brand) => Entry::updateOrCreate($brand, 'brands', 'brand_code', $storeCode));
```

Comment:
My personal preference for code style on this is
```suggestion
        return $brands->filter(fn ($brand) => !blank($brand->name) && !blank($brand->brand_code))
            ->map(fn ($brand) => [
                'title' => $brand->title,
                'brand_code' => $brand->brand_code,
                'path' => $brand->brand_code
            ])
            ->each(fn ($brand) => Entry::updateOrCreate($brand, 'brands', 'brand_code', $storeCode));
```